Update standard-ruleset.rst

PetyaKangalova opened the pull request:
@nosvalds  and @akmiller01  as per the dev page you shared, listing all rules on this page, I have made a few suggested text changes, removing the reference to machine and non machine readable rules, which we are not using and has also caused confusion in the community. Please let me know if there are any additions on your end, especially on using the Standard Ruleset.

PetyaKangalova committed:
Update standard-ruleset.rst

@nosvalds  and @akmiller01  as per the dev page you shared, listing all rules on this page, I have made a few suggested text changes, removing the reference to machine and non machine readable rules, which we are not using and has also caused confusion in the community. Please let me know if there are any additions on your end, especially on using the Standard Ruleset.

nosvalds commented on en/rulesets/standard-ruleset.rst:
```diff
@@ -1,15 +1,13 @@
 Standard Ruleset
 ================
-The Standard Ruleset is a collection of rules that cover most of the requirements of the IATI Standard and that a computer could, in principle, test. Typically, these are rules about the data that cannot be enforced by the schema alone. It covers for example, the ordering of dates, and checks on the format of an iati-identifier.
+The Standard Ruleset is a collection of rules that cover most of the requirements of the IATI Standard and that a computer could, in principle, test. Typically, these are rules about the data that cannot be enforced by the IATI schema alone. It covers for example, the ordering of dates, the use of codelists, percentages adding to 100, checks on the format of an iati-identifier and others.
```
> the use of codelists

codelists are not part of the Rulesets JSON/IATI-Rulesets repo

nosvalds commented on en/rulesets/standard-ruleset.rst:
```diff
@@ -1,15 +1,13 @@
-We call rules in this generic format our "Machine Readable" rules.
+We have created a generic framework that allows us to express many of these rules in a way that machines can understand and humans can interpret by reading a simple error message.
 
 We have written some code to interpret these rules, and hope that they will be useful, for example, for someone that wanted to build a web application that tested people's data against these rules. They should be easy to implement.
```
```suggestion
The IATI Validator V2 uses these rules to programmatically validate IATI documents. The code used to interpret these rules and validate an IATI document are part of the validator GitHub repository. `IATI/js-validator-api <https://github.com/IATI/js-validator-api>`_
```

nosvalds commented on en/rulesets/standard-ruleset.rst:
```diff
@@ -23,25 +21,8 @@ If we take a ``transaction`` from an ``iati-activity`` record as an example, we
 ^^^^^^^^^^^^^^^^^^^^^^
 
-These rules are defined in a `JSON file <https://github.com/IATI/IATI-Rulesets/blob/version-2.03/rulesets/standard.json>`_ as part of our `Single Source of Truth <https://iatistandard.org/en/guidance/developer/ssot/>`_ that can be consumed programatically. The list below has been generated from that source.
+The full list of IATI rules in defined in a `JSON file <https://github.com/IATI/IATI-Rulesets/blob/version-2.03/rulesets/standard.json>`_ as part of our `Single Source of Truth <https://iatistandard.org/en/guidance/developer/ssot/>`_ that can be consumed programatically. The list below has been generated from that source. You can see the error/warning message in plain English and also an id link  ...
```
```suggestion
The full list of IATI rules are defined in a `JSON file <https://github.com/IATI/IATI-Rulesets/blob/version-2.03/rulesets/standard.json>`_ as part of our `Single Source of Truth <https://iatistandard.org/en/guidance/developer/ssot/>`_ that can be consumed programatically. The list below has been generated from that source. You can see the error/warning message in plain English and also an id link that would direct you to the source JSON. You will see in the JSON file there that there is a severity reference. All MUST statements are definited as errors (rules) and all SHOULD statement are defined as warnings (recommended guidance).
```

nosvalds reviewed (changes requested):
Few comments, but good overall!

akmiller01 commented:
Also just noting @PetyaKangalova , with headlines as they are, the rule xpaths come out as h3 where you had asked them to come out as h4. For e.g. `<h3>/iati-activities/iati-activity></h3>`

```
Standard Ruleset
================
```
Needs to be changed to
```
################
Standard Ruleset
################
```

And

```
Using the Standard Ruleset
^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Needs to be changed to:
```
Using the Standard Ruleset
***************************
```

And then the xpaths should come out as h4.

PetyaKangalova committed:
Update en/rulesets/standard-ruleset.rst

Co-authored-by: Nik Osvalds <60047271+nosvalds@users.noreply.github.com>

PetyaKangalova committed:
Update en/rulesets/standard-ruleset.rst

Co-authored-by: Nik Osvalds <60047271+nosvalds@users.noreply.github.com>

PetyaKangalova committed:
Update standard-ruleset.rst

Removed the codelist reference, changed the naming of document to file and added the new headings as per Alex's suggestion.

PetyaKangalova committed:
Update standard-ruleset.rst

Changed  back  to ^ so that rules can be h4.

akmiller01 commented:
Just tried a quick render, looks good to me! 
![Screenshot from 2021-09-08 10-54-51](https://user-images.githubusercontent.com/2836840/132533377-ebfe35f9-83d2-432e-a983-985549d30def.png)
